### PR TITLE
Adding files for the APP-6(B) GDB schema

### PR DESCRIPTION
**Pull request by BobBooth**
This PR contains the csv files for creating the APP-6(B) geodatabase schema.

Addresses #362

### CODE REVIEW

**Comment by BobBooth** [comment]
@dfoll & @csmoore  - please review.

**Comment by csmoore** [comment]
@BobBooth @dfoll - general question - should this include the GDB + lpkx before we review / merge?

File: military-overlay/utils-source-and-test-data/schema-source-files/app6b/name_domains_values/Coded_Domain_LandEquipment.csv [hunk 1]
```diff
@@ -0,0 +1,106 @@
+Name	Value
+GROUND TRACK EQUIPMENT	S-G-E-----
```
Comment (csmoore) [inline military-overlay/utils-source-and-test-data/schema-source-files/app6b/name_domains_values/Coded_Domain_LandEquipment.csv]: This file doesn't look comma delimited (look like tabs) but apparently it still works?

File: military-overlay/utils-source-and-test-data/schema-source-files/app6b/military_feature_schemas/Fields_Activities.csv [hunk 2]
```diff
@@ -0,0 +1,30 @@
...
+hqtffd,String,30,HQ/TF/FD,NULLABLE,HQ_TF_FD,,FALSE,,
+uniquedesignation,String,30,Unique Designation,NULLABLE,,,FALSE,,
+additionalinformation,String,20,Additional Information,NULLABLE,,,FALSE,,
+staffcomment,String,20,Staff Comments,NULLABLE,,,FALSE,,
+higherformation,String,21,Higher Formation,NULLABLE,,,FALSE,,
+reinforced,SmallInteger,2,Reinforced or Reduced,NULLABLE,Reinforced,,FALSE,,
+combateffectiveness,String,5,Combat Effectiveness,NULLABLE,CombatEffectiveness,,FALSE,,
+credibility,SmallInteger,2,Credibility,NULLABLE,Credibility,,FALSE,,
+reliability,String,2,Reliability,NULLABLE,Reliability,,FALSE,,
+iff_sif,String,15,IFF/SIF/AIS,NULLABLE,,,FALSE,,
```
Comment (csmoore) [inline military-overlay/utils-source-and-test-data/schema-source-files/app6b/military_feature_schemas/Fields_Activities.csv]: @jeconley - I remember you IM'd me about this attribute "idmode" (2525) vs. "iff_sif" (app6b) for app6b. It is **iff_sif** in this schema - please let us know if it (or anything else you notice here) should be something different - thanks

**Comment by dfoll** [comment]
@csmoore @BobBooth yes I am inclined towards thinking that when we merge this work, we should have  the gdb, lpkx and all other pieces in place that @csmoore will need to implement 6B in MSE

File: military-overlay/utils-source-and-test-data/schema-source-files/app6b/name_domains_values/Coded_Domain_LandEquipment.csv (hunk 1 above)
Comment (dfoll) [reply military-overlay/utils-source-and-test-data/schema-source-files/app6b/name_domains_values/Coded_Domain_LandEquipment.csv]: fixed with #367

**Comment by csmoore** [comment]
@dfoll - schema table added in https://github.com/Esri/military-symbology/pull/366/commits/7e050f8927977970f46a914567ad5a7493387605 at https://github.com/Esri/military-symbology/blob/bb_362_start_app6b_schema/military-overlay/app6b/README.md so should be good to review now

**Pull request merged**